### ISSUE TITLE
Seems that the results on regression is not ideal as the one shown in the original paper?

### ISSUE THREAD

**Issue by SUZhaoyu** [vfdev-5/CoordConvExperiments#1]
Seems that the results of regression is not good, may I know the reason? I read the original paper's Supplement (S3), however, not much implementation details were covered by the author.

**Comment by vfdev-5** [comment]
Yes, I agree that I can not reproduce the paper results for the regression. @SUZhaoyu if you have a FB account, you can ask for details in their [blog article](https://eng.uber.com/coordconv/)...

**Comment by SUZhaoyu** [comment]
Thanks for your reply and advice @vfdev-5 I have already asked for more implementation details on the blog and I'll keep you posted once got any replies or answers. Btw, I got the similar results as yours (implemented in TF), while most of the predictions trend to squeeze together.

**Comment by vfdev-5** [comment]
@SUZhaoyu I see, good to know that we are several who experience the same issue :)
Do not hesitate to post a link on your implementation here, so I could take a look too

**Comment by ShuangXieIrene** [comment]
I got the some similar result when I try to reproduce the regression experiments. The trainset and testset result are shown in the following.

![trainset_result](https://user-images.githubusercontent.com/23406995/49040804-90ff4300-f191-11e8-9883-f29b8620c672.png)

![testset_result](https://user-images.githubusercontent.com/23406995/49040898-c99f1c80-f191-11e8-9d71-a277804f9ef7.png)
